domains/tim.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Tim is a Dev",
+  "owner": {
+    "username": "xorob0",
+    "email": "xorob0@toum"
+  },
+  "repo": "https://github.com/xorob0/register",
+  "record": {
+    "URL": "https://github.com/tommitchelmore",
```

Comment:
URL redirection along with a cname record will not function correctly. I'd suggest removing it.
```suggestion
```